### PR TITLE
Ap 5325 add plf proceedings

### PR DESCRIPTION
**Pull request by kmahern**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-5325)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`.
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- The standards in the [Git Workflow document on Confluence](https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/4602855954/Git+Workflow) should be followed.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: spec/populators/proceeding_type_populator_spec.rb [hunk 1]
```diff
@@ -12,15 +12,15 @@
     let(:seed_file) { Rails.root.join("db/seed_data/proceeding_types.yml").freeze }
 
     it "create instances from the seed file" do
-      expect { described_class.call }.to change(ProceedingType, :count).by(61)
+      expect { described_class.call }.to change(ProceedingType, :count).by(159)
```
Comment (naseberry) [inline spec/populators/proceeding_type_populator_spec.rb:15]: hmm, I thought there were 95 new proceedings instead of 98.

Comment (naseberry) [inline spec/populators/proceeding_type_populator_spec.rb:15]: I thought there were 95 new proceedings instead on 98?

**Comment by naseberry** [comment]
Too many new proceedings, so I checked against proceedings that should not be added.
Best to get another Dev or BA to check new proceedings

File: spec/populators/proceeding_type_populator_spec.rb (hunk 1 above)
Comment (kmahern) [reply spec/populators/proceeding_type_populator_spec.rb:15]: 🤔 where are you getting the figure of 95 new proceedings from @naseberry ?

Comment (colinbruce) [reply spec/populators/proceeding_type_populator_spec.rb:15]: The source spreadsheet looks a bit all over the place.
* where did the descriptions come from? The SS has PBM01 as `Declaration for overseas adoption-Pub Law-Fam`?
* the count on tab 1 does not match the count on the question matrix

**Pull request closed without merging**